### PR TITLE
feat: Add InputGroup.ts and TextInput.ts

### PR DESCRIPTION
**Pull request by jbrundage**
Signed-off-by: Jaman Brundage <jbrundage372@gmail.com>

### CODE REVIEW

**Comment by GordonSmith** [comment]
Failing travis (syntax errors).

File: packages/layout/src/FlexGrid.ts [hunk 1]
```diff
@@ -58,6 +58,7 @@ export class FlexGrid extends HTMLWidget {
                 });
             })
             ;
+        listItems.exit().remove();
```
Comment (GordonSmith) [inline packages/layout/src/FlexGrid.ts]: Probably not meant to be included?

**Comment by GordonSmith** [comment]
@jbrundage failing unit tests

**Comment by jbrundage** [comment]
@GordonSmith I don't see what's causing the unit tests to fail... at first it was a missing css file, but now I'm not sure what's causing it?

**Comment by GordonSmith** [comment]
@jbrundage can you rebase onto latest?

**Comment by GordonSmith** [comment]
@jbrundage bump

**Comment by jbrundage** [comment]
Closing as #3080 adds better versions of these widgets

**Comment by jbrundage** [comment]
Issued #3104 because of this PR

**Pull request closed without merging**